vetur.config.js

```diff
@@ -1,5 +1,3 @@
-// vetur.config.js
-/** @type {import('vls').VeturConfig} */
```

Comment:
What's the reason for removing this?

Reply:
Added a warning because the type didnt exist.

Didn't bother too much looking for the correct type, as I don't think we need a JSDoc here anyway?